politeiawww/cmd/politeiawwwcli/commands/edituser.go

```diff
@@ -4,6 +4,20 @@ import (
 	"github.com/decred/politeia/politeiawww/api/v1"
 )
 
+// Help message displayed for the command 'politeiawwwcli help edituser'
+var EditUserCmdHelpMsg = `edituser "userid" "action" "reason"
+
+Edit the details for the given user id (admin).
+
+Arguments:
+1. emailnotifications       (uint64, optional)   Whether to notify via emails
+
+Result:
+{
+  "emailnotifications": null
```

Comment:
I would set this as a `uint64` in this example.  So something like:

`"emailnotifications": 1,`

Reply:
Have reworded edituser to:

```
Edit the details for the given user id. Requires admin privileges.

Valid options are:
'0' - User does not receive email notifications   
'1' - User receives email notificaions

Arguments:
1. emailnotifications (uint64, optional)  Email notification setting (bit flag)

Result:
{
  "emailnotifications":  (uint64)  Bit flag
}
{}`
```

Reply:
Updated notification options also, per below updated comment.